src/main/studio/ui/rstextarea/DeleteNextWordAction.java

```diff
@@ -0,0 +1,42 @@
+package studio.ui.rstextarea;
+
+import static javax.swing.text.DefaultEditorKit.deleteNextWordAction;
+
+import java.awt.event.ActionEvent;
+import javax.swing.UIManager;
+import javax.swing.text.BadLocationException;
+import javax.swing.text.JTextComponent;
+import javax.swing.text.TextAction;
+import javax.swing.text.Utilities;
+import org.fife.ui.rtextarea.RTextArea;
+import org.fife.ui.rtextarea.RecordableTextAction;
+import studio.ui.EditorPane;
+
+public class DeleteNextWordAction extends TextAction {
```

Comment:
All actions in RSTA are extended from RecordableTextAction. Not sure if it is possible to record macros now. But if we add such functionality in the future, likely we will need to have the action derived from RecordableTextAction and override getMacroID.

Reply:
Fair enough - updated the PR